docs: Add README.md (#14)
contane/charts: README.md

> # Contane Helm Charts

Reviewer: Please add a single newline after every heading

> Welcome to the Contane Helm Charts repository! Here, you'll find a collection of Helm Charts for all the open-source projects developed by Contane. These charts are designed to simplify the deployment and management of applications on Kubernetes clusters.

Reviewer: This is slightly inaccurate - we cannot provide a Helm chart for *all* open-source projects, given that not all projects are applications (e.g., docker-yamllint). I also think we can make it more concise.

```suggestion
A collection of Helm charts for open-source applications developed by Contane, designed to simplify the deployment on Kubernetes.
```

## Features
- **Ready to Use**: All Helm Charts in this repository are pre-configured for immediate use. They can be deployed with default settings, making it easy to get started.
- **Customizable**: For production environments, we highly recommend configuring the charts to suit your specific needs. Each chart provides various configuration options to tailor the deployment.
- **Consistent Updates**: Our charts are regularly updated to ensure compatibility with the latest versions of Kubernetes and Helm, as well as to include new features and bug fixes.

## Prerequisites
Before you begin, ensure you have the following:

> - **A Running Kubernetes Cluster**: A functional Kubernetes cluster is required to deploy the Helm Charts. You can set up a cluster using Minikube, Kubernetes in Docker (Kind), or any cloud provider like GKE, EKS, or AKS.
> - **Helm 3.x Installed**: Ensure that you have Helm version 3.x installed on your local machine. You can follow the official Helm installation guide for detailed instructions.

Reviewer: I don't think we should - or need to - provide specific guidance here. This repository is highly unlikely to be the starting point for someone who never worked with Kubernetes. I'm aware that Bitnami has install guides for Kubernetes, but those focus on using their charts specifically, so unless we can provide additional value, I'd say we simplify this.

However, we could link to the respective official guides for Kubernetes and Helm?

```suggestion
- Kubernetes: refer to the Kubernetes [getting started guide](https://kubernetes.io/docs/setup/)
- Helm 3.x: refer to the [Helm install guide](https://helm.sh/docs/intro/install/)
```

## Installation of Contane Helm Charts

> Installing a Contane Helm Chart is straightforward. Follow these steps to get started:

Reviewer: This is merely an opinion, and has the potential to alienate users who have trouble using our charts from asking for help.

```suggestion
Follow these steps to get started:
```

> ### Using the Helm Chart Repository

Reviewer: Can we put the OCI registry section first?

> 1. **Add the Repository**:

Reviewer: ```suggestion
1. **Add the repository**:
```

    ```bash

>     helm repo add contane https://ghcr.io/contane/charts

Reviewer: ```suggestion
    helm repo add contane https://contane.github.io/charts
```

    helm repo update
    ```

> 2. **Install a Chart**:

Reviewer: ```suggestion
2. **Install a chart**:
```

    Replace `<chart-name>` with the name of the chart:
    ```bash
    helm install my-custom-release contane/<chart-name>
    ```

> 3. **Verify the Installation**:

Reviewer: ```suggestion
3. **Verify the installation**:
```

    ```bash
    helm list
    kubectl get all -l app.kubernetes.io/instance=my-custom-release
    ```

### Using the OCI Registry

> 1. **Install a Chart**:

Reviewer: ```suggestion
1. **Install a chart**:
```

    Replace `<chart-name>` with the name of the chart:
    ```bash
    helm install my-custom-release oci://ghcr.io/contane/charts/<chart-name>
    ```

> 2. **Verify the Installation**:

Reviewer: ```suggestion
2. **Verify the installation**:
```

    ```bash
    helm list
    kubectl get all -l app.kubernetes.io/instance=my-custom-release
    ```

## Configuration

> While the charts come with sensible defaults, you can customize their behavior by providing your own values. To do this, you can create a `values.yaml` file and pass it during installation:

Reviewer: At the start, we are recommending to always customize production deployments, so the tone of this section should reflect that.

```suggestion
You can customize the charts by creating a `values.yaml` file and providing it during installation:
```

```bash
helm install my-custom-release contane/<chart-name> -f values.yaml
```

Refer to the `README.md` file in each chart's directory for detailed information on the available configuration options.

## Upgrading Charts
To upgrade an existing release to a newer version of the chart, use the following command:

```bash
helm upgrade my-custom-release contane/<chart-name>
```

## Uninstallation
To uninstall a release and remove all associated Kubernetes resources, use:

```bash
helm uninstall my-custom-release
```

> ## Contributing
> We welcome contributions from the community! If you have suggestions, find a bug, or want to add new features, please open an issue or submit a pull request.
> 
> ## License
> This repository is licensed under the Apache-2.0 license. Feel free to use and modify the charts as per the terms of the license.
> 
> ## Support
> If you encounter any issues or have any questions, please open an issue in this repository. Our team and the community are here to help!
> 
> Thank you for using Contane Helm Charts!

Reviewer: I don't think the README needs its own section regarding the license, since that is available in the LICENSE file, and GitHub shows it prominently in their UI as well.

Additionally, the "Support" section seems to overlap 100% with "Contributing" and can IMHO be removed.

```suggestion
## Contributing

We welcome contributions from the community! If you have suggestions, find a bug, or want to add new features, please open an issue or submit a pull request.
```
